templates/shared/_promoted_roles.html

```diff
@@ -1,46 +1,50 @@
 <section class="p-section">
   {% if featured_jobs %}
-  <div class="u-fixed-width">
-    <hr class="p-rule">
-  </div>
-  <div class="row--25-75">
-    <div class="col-3 col-medium-3">
-      <div class="p-block">
-        <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">Featured Roles</h2>
-      </div>
+    <div class="u-fixed-width">
+      <hr class="p-rule" />
     </div>
-    <div class="col-9 col-medium-3">
-      <div class="row">
-      {% for job in featured_jobs %}
-      {% if loop.index < 10 %}
-        <div class="col-3">
-          {% if loop.index < 4 %}
-            <hr class="p-rule u-hide--large {% if loop.index == 1 %}u-hide--medium{% endif %}">
-          {% else %}
-            <hr class="p-rule">
-          {% endif %}
-          <p><a class="p-featured__url p-card--content" href="/careers/{{job.id}}" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Featured job', 'eventAction' : 'Click', 'eventLabel' : '{{ job.title }}', 'eventValue' : undefined });">{{ job.title }}</a></p>
+    <div class="row--25-75">
+      <div class="col-3 col-medium-3">
```

Comment:
Not sure what was the intention here (I know it's not from this PR), but inside `row--25-75` columns don't need (and technically should not) be numbered by width. It should just be `<div class="col">` as it's the row that determines the spacing.

```suggestion
      <div class="col">
```

Reply:
Thanks for catching! I'd been trying to find things like this throughout the other removal prs but admittedly rushed through this one and missed it 😅